src/components/editor/PropertyTemplateOutline.jsx

```diff
@@ -93,115 +96,136 @@ export class PropertyTemplateOutline extends Component {
     }
   }
 
-  handleComponentCase = (property) => {
```

Comment:
since we use "handle" for things like javascript button presses and clicks and all, it seemed prudent to me to name this differently.